Fauna-RPG/Core/Battler.gd

```diff
@@ -25,15 +25,15 @@ func _ready() -> void:
 
 
 func _summon_enemies() -> void:
-	for enemy_index in enemies_to_summon.size():
+	for enemy_index: int in enemies_to_summon.size():
```

Comment:
Index is used only as a way to iterate over the list, can directly iterate over the elements instead

Reply:
Seems to be the same as [this comment I made](https://github.com/Saplings-Projects/1M_sub/pull/1#discussion_r1412276583), so for the same reason my comment is irrelevant here; because we do use it later